nyxl-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,416 @@
+@import "portlet_decorator";
+
+@import "portlet_hello_world";
+
+@import "header";
+
+body {
+	background-color: $bg-color;
+	font-family: $body-font;
+}
+
+a {
+	transition: 0.3s ease-in-out;
+	// white-space: nowrap;
+}
+
+.container-fluid-1280 .row,
+.portlet {
+	margin: 0;
+}
+
+.btn, .btn-default {
+	@include btnBackground($lightblue-color);
```

Comment:
Line break needed to space out mixin and the rest of the CSS properties.  See line 27-29, you did it correctly there.